m4_sessions/1806-inning/group6.md

```diff
@@ -1,33 +1,42 @@
 ## Group Members:
-
+Kelly Schroeder, Helen Dechat, Sage Vogt, David 'Doc' Daugherty
 
 
 ## Name of Session: 
-
+ VR & AR: How they will affect our lives in the future.
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+The ways in which AR and VR will affect future technologies and how those technologies will impact society regarding industry, healthcare, education, and entertainment. 
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+(What will students learn/leave with?): Excited AF about AR & VR. With a better understanding, and hopefully increase interest, in the coming wave of AR & VR technologies.
 
 ## Session Outline 
 
 (Please include who is facilitating and how much time you are going to spend in each section):
+     15 minutes on each of the 4 sections with 10 minute Q&A at the end.
+     Industry: Kelly
+     Healthcare: Helen
+     Education: Sage
+     Entertainment: Sage
 
-
-
+     
      Best practices:  No section should be longer than 20 minutes
      Create a session that YOU would want to attend
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
     
 ## Recommended Trajectory 
-## Introductions 
+Where the technology is now, current advances and hurdles, expected growth and outcome, dangers of the technology.
+## Introductions
+We'll introduce ourselves and why we chose this topic.
 ## Intro to new material
+Brief description of what we will be discussing, why we'll be discussing, and set an expectation for the students as to how we will structure the talk.
 ## Small Group Activity
+Assuming we can find an appropriate app, we would like to have the students download a VR/AR app prior to the session, and we will do an involved activity with the app. 
 ## Closing Discussion
+What is needed for AR & VR to take the next step into mainstream adoption?
```

Comment:
What are the benefits of AR and VR in your life?  What is most exciting about VR and AR?  What are the dangers of VR and AR?